SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfigurationLoader.java

```diff
@@ -47,6 +47,7 @@
   public static final String TAIL_LOG_LINES_TO_SAVE = "executor.service.log.tail.lines.to.save";
   public static final String TAIL_LOG_FILENAME = "executor.service.log.tail.file.name";
 
+  public static final String S3_FILES_TO_BACKUP = "executor.s3.uploader.extras.files";
```

Comment:
I was copying the format of https://github.com/HubSpot/Singularity/blob/persist_more_logs/SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfigurationLoader.java#L45 but I agree that additional is a better word. Which would you prefer?